From a0df8d1cddccde9368b075d19347632934bfdafc Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Mon, 20 Jan 2014 11:07:37 +1000
Subject: [PATCH] conf: add N-Trig DuoSense to our matched devices

That device has a pen and an eraser tool, as well as a touch device. Handle it
with the wacom driver by default, the evdev driver isn't quite up to scratch
here.

This also fixes a missing match in the .fdi file for the base N-Trig devices -
just in case.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 conf/50-wacom.conf | 2 +-
 conf/wacom.fdi     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/conf/50-wacom.conf b/conf/50-wacom.conf
index 567b900..21f75e6 100644
--- a/conf/50-wacom.conf
+++ b/conf/50-wacom.conf
@@ -29,7 +29,7 @@ EndSection
 # N-Trig Duosense Electromagnetic Digitizer
 Section "InputClass"
 	Identifier "Wacom N-Trig class"
-	MatchProduct "HID 1b96:0001|N-Trig Pen"
+	MatchProduct "HID 1b96:0001|N-Trig Pen|N-trig DuoSense"
 	MatchDevicePath "/dev/input/event*"
 	Driver "wacom"
 	Option "Button2" "3"
diff --git a/conf/wacom.fdi b/conf/wacom.fdi
index 70304d2..dd623a4 100644
--- a/conf/wacom.fdi
+++ b/conf/wacom.fdi
@@ -15,7 +15,7 @@
       </match>
     </match>
     <!-- N-Trig Duosense Electromagnetic Digitizer -->
-    <match key="info.product" contains="HID 1b96:0001">
+    <match key="info.product" contains_outof="HID 1b96:0001;N-Trig Pen;N-trig DuoSense">
       <match key="info.parent" contains="if0">
        <merge key="input.x11_driver" type="string">wacom</merge>
       </match>
-- 
1.8.4.2

